install/kubernetes/cilium/charts/hubble-cli/templates/daemonset.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.global.hubble.enabled }}
+{{- if .Values.hubble-cli.enabled }}
```

Comment:
This changes the test condition (hubble.enabled vs hubble-cli enabled). What's the rationale for this change?

I personally think we can also remove this if. In hubble-relay and ui we have the validation.yaml instead.

Reply:
The flag for `enabled=true` controls whether or not the hubble-cli deployment manifest is deployed in the cluster. It is a control for the sub-chart. I do see the need to change the test in addition to this.